### PR TITLE
Updating Import for Sinks, from worker -> workflow

### PR DESCRIPTION
**Pull request by ELares**
## What does this PR do?
Minor bug fix: Sinks is not part of `@temporalio/worker`, but rather `@temporalio/workflow`
## Notes to reviewers
Minor bug fix in imports for Sink.

### CODE REVIEW

**Review by bergundy — APPROVED** [review]
Thanks!

**Comment by bergundy** [comment]
@flossypurse can you please help fix the build here?

**Comment by flossypurse** [comment]
> @flossypurse can you please help fix the build here?

You can merge this if you want and ignore the build issue -

I noticed this issued and fixed it when I merged in master in this PR: https://github.com/temporalio/documentation/pull/728/files#diff-16b6401519950c5f36e2548e217df9174771c97bbfb0c8ead59906fe108d28feR14

Am waiting on one more review - then it will land and fix it.

**Review by sw-yx — APPROVED** [review]

**Pull request merged**
